frontend/src/store/app/actions.ts

```diff
@@ -1,8 +1,8 @@
+import { PBMessage } from '../controller/types';
```

Comment:
app/types shouldn't depend on the existence of controller/types, and ideally it wouldn't depend on the existence of P message.